src/Java.Interop/Java.Interop/JniRuntime.JniMarshalMemberBuilder.cs

```diff
@@ -48,8 +48,8 @@ partial class CreationOptions {
 			var t   = jie.GetType ("Java.Interop.MarshalMemberBuilder");
 			if (t == null)
 				throw new InvalidOperationException ("Could not find Java.Interop.MarshalMemberBuilder from Java.Interop.Export.dll!");
-			var b   = (JniMarshalMemberBuilder) Activator.CreateInstance (t);
-			marshalMemberBuilder    = SetRuntime (b);
+			var b   = (JniMarshalMemberBuilder?) Activator.CreateInstance (t);
```

Comment:
Not sure I understand this change, as [`Activator.CreateInstance()`](https://docs.microsoft.com/en-us/dotnet/api/system.activator.createinstance?view=netcore-3.1#System_Activator_CreateInstance_System_Type_) is only documented as returning `null` when `type` is a `Nullable<T>`, which won't be the case here.

Reply:
I guess I prefer the approach taken later in this PR:

```csharp
var b   = (JniMarshalMemberBuilder) Activator.CreateInstance (t)!;
```